### ISSUE TITLE
best approach to translate and/or make multilignual this theme?

### ISSUE THREAD

**Issue by klud1** [jpescador/hugo-future-imperfect#39]
I'm starting a blog, and love this theme, but i want to write content in spanish and english, tried to translate by the theme following the hugo guides, but not everything is translated; some parts of the theme are still in english

**Comment by pacollins** [comment]
@klud1 - What all are you trying to translate - content or main theme structures (I realize you posted this in April).

**Comment by pacollins** [comment]
@statnmap - any chance you could update to this version and create a PR for your implementation?

**Comment by statnmap** [comment]
Yes. You can have a look at my PR #58 and modify it if necessary.

**Comment by pacollins** [comment]
@untalpierre #58 is starting the process on this. I have fixed all the merge requests, if you could review that this is working for you, that would help.

**Comment by henningsway** [comment]
@statnmap is there a place to see how you approached this problem? (Would also be relevant to me right now)

**Comment by statnmap** [comment]
This is ongoing work that you can see here https://github.com/jpescador/hugo-future-imperfect/pull/58.
